clang/lib/CodeGen/CGRecordLayoutBuilder.cpp

```diff
@@ -950,18 +948,20 @@ void CGRecordLowering::calculateZeroInit() {
 }
 
 // Verify accumulateBitfields computed the correct storage representations.
-void CGRecordLowering::checkBitfieldClipping() const {
+void CGRecordLowering::checkBitfieldClipping(
+    bool IsNonVirtualBaseType LLVM_ATTRIBUTE_UNUSED) const {
```

Comment:
The doc of LLVM_ATTRIBUTE_UNUSED doesn't suggest variable uses.

Since we use `-Wno-unused-parameter` and `IsNonVirtualBaseType` is used in !NDEBUG code paths, we can just use `(bool IsNonVirtualBaseType)`